src/IssueViz.Security.UnitTests/Taint/TaintIssuesSynchronizerTests.cs

```diff
@@ -569,17 +638,21 @@ private void SetupAnalysisInformation(Mock<ISonarQubeService> sonarQubeService,
                     new SonarQubeProjectBranch(Guid.NewGuid().ToString(), false, DateTimeOffset.MinValue)
                 };
 
-            sonarQubeService.Setup(x => x.GetProjectBranchesAsync(SharedProjectKey, CancellationToken.None))
+            sonarQubeService.Setup(x => x.GetProjectBranchesAsync(projectKey, CancellationToken.None))
                 .ReturnsAsync(projectBranches);
         }
 
         private void SetupTaintIssues(Mock<ISonarQubeService> sonarQubeService, params SonarQubeIssue[] issues)
-            => SetupTaintIssues(sonarQubeService, null, issues);
+        {
+            sonarQubeService
+                .Setup(x => x.GetTaintVulnerabilitiesAsync(It.IsAny<string>(), It.IsAny<string>(), CancellationToken.None))
+                .ReturnsAsync(issues);
+        }
 
-        private void SetupTaintIssues(Mock<ISonarQubeService> sonarQubeService, string branch = null, params SonarQubeIssue[] issues)
+        private void SetupTaintIssues(Mock<ISonarQubeService> sonarQubeService, string projectKey, string branch, params SonarQubeIssue[] issues)
```

Comment:
... and this overload is used when the caller does care what the project/branch parameters are.